package.json

```diff
@@ -31,7 +31,8 @@
     "lint-js": "xo",
     "test": "npm run lint-html && npm run lint-css && npm run lint-js",
     "test-e2e": "gulp test-e2e",
-    "dev": "gulp dev & nodemon --watch app/main --watch app/renderer --exec 'npm test' -e html,css,js",
+    "compile": "gulp compile",
+    "dev": "gulp dev & nodemon --watch app/main --watch app/renderer --exec 'npm test' ",
```

Comment:
A gentle reminder that `dev` script was broken when I last checked.

Reply:
Fixed this. I think the nodemon part was not required. The processes are automatically reloaded from the gulp scripts in gulpfile.js .